src/Components/Detail/Detail.scss

```
@@ -0,0 +1,65 @@
+.detailProduct {
+  -webkit-font-smoothing: antialiased;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+  flex-direction: row;
+  font-family: 'Suisse Regular', sans-serif;
+
+  .detailImg img {
+    width: 600px;
+    height: 400px;
+  }
+
+  .detailBox {
+    color: rgb(51, 51, 51);
+    font-size: 11px;
+    width: 350px;
+    .category {
+      ul {
+        display: flex;
+        margin: {
+          right: 30px;
+          bottom: 20px;
+        }
+      }
+    }
+
+    .detailNameExplain {
+      border-bottom: 2px solid black;
+      .detailName {
+        font-size: 25px;
+        margin-bottom: 20px;
+      }
+      .detailExplain {
+        margin-bottom: 20px;
+      }
+    }
+
+    .detailInfo {
+      margin-top: 15px;
+      p {
+        margin-bottom: 10px;
+        &:last-child {
+          color: #686868;
+        }
+      }
+      li {
+        border-bottom: 1px solid #d6d5cc;
+        margin-bottom: 15px;
+        &:last-child {
+          border-bottom: none;
+        }
+      }
```

```suggestion
      p {
        margin-bottom: 10px;
        
        &:last-child {
          color: #686868;
        }
      }
      
      li {
        border-bottom: 1px solid #d6d5cc;
        margin-bottom: 15px;
        
        &:last-child {
          border-bottom: none;
        }
      }
```
가독성을 위해 셀럭터 시작할 때는 한 줄 띄워주시기 바랍니다.

---

수정완료하였습니다!